### PR TITLE
Verified PeriodicAzureCookieSetter renders under the correct conditions.

### PR DESCRIPTION
**Pull request by n8landolt**
### What
- Add test coverage to the Periodic Azure Cookie Setter functionality.

### Why
- There was a prod incident which stemmed from the cookie not being set correctly.

### Testing strategy
- [ ] run the tests
- [ ] verify the language in the test cases

### CODE REVIEW

File: src/analysis/modals/CloudEnvironmentModal.test.ts [hunk 1]
```diff
@@ -143,6 +154,64 @@ const CloudEnvironmentModalDefaultProps: any = {
   filterForTool: undefined,
 };
 
+const cromwellRunning = {
```
Comment (jdcanas) [inline src/analysis/modals/CloudEnvironmentModal.test.ts]: you should type this test data as an app to ensure it adheres to the contract as it evolves

**Review by jdcanas — APPROVED** [review]
This looks great, very thorough. There was definitely a sneaky amount of logic here to test

File: src/analysis/PeriodicAzureCookieSetter.test.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  it('should call usePollingEffect', async () => {
+    // Arrange
+    await render(h(PeriodicAzureCookieSetter));
+    // Act
+
+    // Assert
+    expect(usePollingEffect).toHaveBeenCalled();
+  });
+
+  it('should call usePollingEffect', async () => {
```
Comment (lucymcnatt) [inline src/analysis/PeriodicAzureCookieSetter.test.ts]: these 2 tests are called the same thing, should the second also include `withErrorIgnoring`?

**Review by lucymcnatt — APPROVED** [review]
small nit, but otherwise looks good!

**Pull request merged**